package.json

```diff
@@ -24,7 +24,7 @@
     "bracketSpacing": true,
     "semi": false,
     "singleQuote": true,
-    "trailingComma": "es5"
+    "trailingComma": "none"
```

Comment:
@erikdstock was able to fix with a [`// disabled`](https://github.com/artsy/reaction/pull/636#discussion_r182258865) comment

Reply:
I'm not sure exactly -- @erikdstock what were your precise steps? (I think the full comment is `// prettier-ignore` if indeed it is prettier and not TSLint.)

But yeah, we should just consider the trailing comma error a tooling bug that will eventually be resolved and disable autofixes in those area since trailing commas are so good for refactoring / moving around code.